### ISSUE TITLE
Practitioners pouch problem with Ritual of The Deep Ocean

### ISSUE THREAD

**Issue by Sarithule** [Mithion/Mana-And-Artifice#863]
**Describe the bug**
When the attempting to activate the Ritual of The Deep Ocean with a practitioners pouch, it claims that "The Pouch does not have enough items to place the ritual"
The pouch contains plenty of Sand, Seagrass, Raw Cod, Wizard Chalk, and Purified Vintium dust

**To Reproduce**
Store the ritual of the deep ocean in a practitioners pouch.
Attempt to activate the ritual of the deep ocean with sand, and raw cod and all other components required for the ritual in the pouch.

**Expected behavior**
ritual should activate

**Screenshots**
![2024-07-11_16 30 09](https://github.com/user-attachments/assets/e3690568-464a-4036-97b6-83800d3abfeb)
![2024-07-11_16 30 19](https://github.com/user-attachments/assets/1fbe9ead-1e36-441d-841b-7a276f439153)

**Version Information (please complete the following information):**
 - Mod Version: [e.g. 3.0.0.19]
 - Forge Version [47.3.0]

**Comment by Sarithule** [comment]
Looks like the Tool Tip is requesting both raw cod AND cooked cod?
![2024-07-11_16 59 17](https://github.com/user-attachments/assets/d7f76607-632a-4437-a6e8-770c37400a01)

**Comment by Mithion** [comment]
It uses the EXACT reagents you stored it with in the EXACT places, and it will not substitute anything, no tags, no nothing - try re-storing it using only raw cod.

**Comment by Sarithule** [comment]
Yup, that was it. I think I made the original ritual that I stored in the pouch by using the empty hand technique that auto grabs items from the inventory and it must have grabbed both raw and cooked cod. They look so similar that I didn't even notice. Thank you!
